.travis.yml

```diff
@@ -51,27 +51,8 @@ script:
   # run the full testsuite
   - if [ $TEST == "testsuite" ]; then docker exec sardana-test sardanatestsuite; fi
   # build docs
-  - if [ $TEST == "doc" ]; then
-      docker exec -t sardana-test /bin/bash -c "cd /sardana ; sphinx-build -W doc/source/ build/sphinx/html" ;
-    fi
-
-  # deploy sphinx docs to sardana-doc repo if we are on upstream
-  - if [[ $TEST == "doc" && $TRAVIS_REPO_SLUG == "sardana-org/sardana" ]]; then
-      pip install doctr ;
-      docker exec sardana-test /bin/bash -c "touch /sardana/build/sphinx/html/.nojekyll" ;
-      if [[ $TRAVIS_BRANCH == "develop" ]]; then
-        doctr deploy . ;
-      else
-        doctr deploy "v-$TRAVIS_BRANCH"  ;
-      fi;
-    fi
-
-doctr:
-  key-path : ci/github_deploy_key.enc
-  deploy-repo : sardana-org/sardana-doc
-  require-master: false
-  sync: true
-  built-docs: build/sphinx/html/
+  - if [ $TEST == "doc" ]; then docker exec -t sardana-test /bin/bash -c "cd /sardana ; sphinx-build -W doc/source/ build/sphinx/html" ; fi
+  - if [ $TEST == "doc" ]; then docker exec -t sardana-test /bin/bash -c "touch /sardana/build/sphinx/html/.nojekyll" ; fi
```

Comment:
This is a minor thing, but I think that forcing the .nojekyll file is no longer needed because it can be set as a project option (not 100% sure). Checking it is something that I had in my low-priority TO-DO list... maybe we can try it now.

Reply:
I looked in the GitHub repo (in the GitHub Pages zone) but could not find this parameter. Now I would not block this PR for that.